test/unit/utils/api-request.spec.ts

```diff
@@ -847,13 +888,9 @@ describe('HttpClient', () => {
         'content-type': 'application/json',
       });
     mockedRequests.push(scope);
-    const client = new HttpClient({
-      maxRetries: 4,
-      backOffFactor: 0.5,
-      maxDelayInMillis: 60 * 1000,
-      statusCodes: [503],
-    });
+    const client = new HttpClient();
```

Comment:
Should you pass explicit test retry config with backoff factor on, here and elsewhere?

Reply:
Used `defaultRetryConfig()` directly.